cabal.project

```diff
@@ -156,58 +268,14 @@ source-repository-package
     subdir: command-line
             core
 
--- Should follow cardano-wallet.
-source-repository-package
-    type: git
-    location: https://github.com/input-output-hk/cardano-base
-    tag: 0f3a867493059e650cda69e20a5cbf1ace289a57
-    subdir:
-            base-deriving-via
-            binary
-            binary/test
-            cardano-crypto-class
-            cardano-crypto-praos
-            cardano-crypto-tests
-            orphans-deriving-via
-            measures
-            strict-containers
-            slotting
-
--- Should follow cardano-wallet.
-source-repository-package
-    type: git
-    location: https://github.com/input-output-hk/cardano-crypto
-    tag: f73079303f663e028288f9f4a9e08bcca39a923e
-
--- Should follow cardano-node.
--- But in case there are failures with the plutus version, update to the latest
--- commit hash of the release/1.0.0 plutus branch.
-source-repository-package
-  type: git
-  location: https://github.com/input-output-hk/plutus
-  tag: a56c96598b4b25c9e28215214d25189331087244
-  subdir:
-    plutus-core
-    plutus-ledger-api
-    plutus-tx
-    plutus-tx-plugin
-    prettyprinter-configurable
-    stubs/plutus-ghc-stub
-    word-array
-
--- Should follow cardano-node
-source-repository-package
-  type: git
-  location: https://github.com/input-output-hk/ekg-forward
-  tag: 297cd9db5074339a2fb2e5ae7d0780debb670c63
-
 -- Should follow cardano-node
 source-repository-package
   type: git
   location: https://github.com/input-output-hk/cardano-config
   tag: 1646e9167fab36c0bff82317743b96efa2d3adaa
 
--- Should follow cardano-wallet.
+-- This is needed because we rely on an unreleased feature
+-- https://github.com/input-output-hk/cardano-ledger/pull/3111
```

Comment:
:see_no_evil:

Reply:
Adding insult to the injury, cardano-ledger just got a new release out and that feature is not included :sob:

Reply:
Well, their new release is based on functionality that was done on August. Not really up to date. Don't know why though.